src/components/Editor/Board/Canvas/Shape.ts

```diff
@@ -0,0 +1,34 @@
+// TODO yorkie-sdk-js ArrayProxy
```

Comment:
To remember the creator and responsibilities of TODO, please leave a TODO comment in the pattern below.

```suggestion
// TODO(ppeeou): yorkie-sdk-js ArrayProxy
```